<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>creditcalc</class>
 <widget class="QDialog" name="creditcalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(203, 203, 203);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>    Сумма Кредита:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_SumCredit">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Срок Кредита(в месецах):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_TimeCredit">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Процентная ставка:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Procent">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Monospace</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Тип ежемесечных платежей:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_annuity">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Аннуитетные</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_differentiated">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Дифференцированные</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_calculate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>471</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 76, 16);</string>
   </property>
   <property name="text">
    <string>Расчитать</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>471</width>
     <height>89</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>  Ежемесячный платеж:</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_monthlyPayment">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Начисленные проценты:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_overpayment">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>11</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>        Oбщий платеж:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_totalPayment">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
  background-color: rgb(255, 255, 255);
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}
</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget { border: 1px solid black; }
QTableWidget::item { padding: 5px; border: 1px solid gray; }
QTableWidget::item:selected { background-color: lightgray; }
QTableWidget::item:!selected { background-color: white; }
QTableWidget::item:!selected:hover { background-color: lightgray; }
QTableWidget QHeaderView::section { background-color: lightgray; }
QTableWidget QHeaderView::section:checked { background-color: lightgray; }
QTableWidget QHeaderView { border: 1px solid gray; }</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>№ Платежа</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> Сумма платежа</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>          1</string>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
